content/en/tracing/configure_data_security/_index.md

```diff
@@ -13,31 +13,127 @@ aliases:
+| URI userinfo        |                                 |                                 |
+
+{{% /tab %}}
+
+{{% tab "PHP" %}}
+
+| Category            | Collected                       | Obfuscated                      |
+|:--------------------|:-------------------------------:|:-------------------------------:|
+| Name                |                                 |                                 |
+| Email               |                                 |                                 |
+| Client IP           |                                 |                                 |
+| Database statements | <i class="icon-check-bold"></i> | <i class="icon-check-bold"></i> |
+| Geographic location |                                 |                                 |
+| URI parameters      | <i class="icon-check-bold"></i> | <i class="icon-check-bold"></i> |
+| URI userinfo        | <i class="icon-check-bold"></i> | <i class="icon-check-bold"></i> |
+
+{{% /tab %}}
+
+{{% tab "Python" %}}
+
+**Note:** Client IP, geographic location, and URI parameters are not collected by default and must be enabled.
+
+| Category            | Collected                       | Obfuscated                      |
+|:--------------------|:-------------------------------:|:-------------------------------:|
+| Name                | <i class="icon-check-bold"></i> | <i class="icon-check-bold"></i> |
+| Email               | <i class="icon-check-bold"></i> | <i class="icon-check-bold"></i> |
+| Client IP           | <i class="icon-check-bold"></i> | <i class="icon-check-bold"></i> |
+| Database statements | <i class="icon-check-bold"></i> | <i class="icon-check-bold"></i> |
+| Geographic location | <i class="icon-check-bold"></i> |                                 |
+| URI parameters      | <i class="icon-check-bold"></i> | <i class="icon-check-bold"></i> |
+| URI userinfo        |                                 |                                 |
+
+[1]: /tracing/trace_collection/compatibility/python/#datastore-compatibility
+{{% /tab %}}
+
+{{% /tabs %}}
+
+
+The Application Security Management product also collects the following HTTP data in traces (obfuscation is applied):
+
+| Category     | Description                                                                                                                               |
+|:-------------|:------------------------------------------------------------------------------------------------------------------------------------------|
+| HTTP body    | The HTTP request or response payload, which may contain personal information that can be reasonable linked to an individual or household. |
+| HTTP cookies | The Set-Cookie HTTP response header sent by the server that can be sent by the user's browser in the HTTP request header.                 |
+| HTTP headers | The HTTP request or response headers.                                                                                                     |
```

Comment:
I rather think this should link to the ASM documentation at our data privacy section at https://docs.datadoghq.com/security/application_security/how-appsec-works/#data-privacy

```suggestion
You can also read more about data privacy for the Application Security Management product at /security/application_security/how-appsec-works/#data-privacy
```

But not even sure you need this here.

Reply:
That was my next feedback: what is said is not exactly true, and explaining in my link: we don't send those, but rather the subset that matched our security rules. This subset is where the obfuscation logic gets executed. But we never send the body, cookies, or headers as stated here. So I think our documentation state the right level of details for this.

Reply:
I understood that the subparts of the body, cookies, and headers that match security rules are sent after obfuscation, i.e., an HTTP header that matched a security rule would be obfuscated and sent with the trace for details on a security event. Is that accurate?

Reply:
Made the changes suggested by @Julio-Guerra, thanks!